builder/dockerfile/builder.go

```diff
@@ -324,6 +324,9 @@ func (b *Builder) build(stdout io.Writer, stderr io.Writer, out io.Writer) (stri
 	}
 
 	fmt.Fprintf(b.Stdout, "Successfully built %s\n", shortImgID)
+	for _, rt := range repoAndTags {
```

Comment:
nit: I think the error handling makes more sense if we move the "Successfully built" before the tagging and combine the loops. In the future builder shouldn't do the tagging at all and should leave it to the API layer(or client).